### DISCUSSION TITLE
Copilot Example - Are slash commands only user invokable?

### DISCUSSION THREAD

**Discussion by InTheCloudDan** [microsoft/vscode-discussions#957] [Extension Development QnA] [answered]
Trying to understand if this line https://github.com/microsoft/vscode-extension-samples/blob/main/chat-agent-sample/src/vscode.proposed.chatAgents2.d.ts#L244

The {@link ChatAgentSlashCommand slash command} that was selected for this request should be ...by the user. Or if there is some way for the model parsing the request to determine a slash command would be useful and invoke it?

**Comment by isidorn** [comment, marked as answer]
They are always user invoked :)
Let me know if you have other questions.
